tests/system/utils.py

```diff
@@ -38,3 +40,108 @@ def get_resource(resource):
                 raise Exception
         except Exception:
             raise DCOSException("Can't read from resource: {0}. Please check that it exists.".format(resource))
+
+
+class FaultDomain:
+    """
+    High-level representation of a fault domain
+    """
+    def __init__(self, config):
+        # Make sure config is a dict
+        if not type(config) is dict:
```

Comment:
This should go. Python uses duck typing. As long as `config` offers a proper `get` method it's fine. If not the caller will see the proper exception.

Reply:
I think it's purpose was not very clear. I updated the comment. 
Please let me know if you still have questions on this.

Reply:
Also, do you know any concrete example were we are using something that is not a `dict` but uses the `.get` method in our codebase?